agr_api/src/main/java/org/alliancegenome/api/controller/GeneController.java

```diff
@@ -442,7 +442,7 @@ public Response getMolecularInteractionsDownload(String id, String sortBy, Strin
 		Pagination pagination = new Pagination(1, 150000, sortBy, asc);
 		pagination.addFilterOption("geneMolecularInteraction.interactorAType.name.keyword", moleculeType);
 		pagination.addFilterOption("geneMolecularInteraction.geneGeneAssociationObject.geneSymbol.displayText", interactorGeneSymbol);
-		pagination.addFilterOption("geneMolecularInteraction.interactionIdORgeneMolecularInteraction.aggregationDatabase.nameORgeneMolecularInteraction.interactionSource.name", source);
+		pagination.addFilterOption("geneMolecularInteraction.interactionIdORgeneMolecularInteraction.aggregationDatabase.nameORgeneMolecularInteraction.interactionSource.nameORgeneMolecularInteraction.crossReferences.displayName", source);
```

Comment:
@markquintontulloch I am guessing that `interactionIdORgeneMolecularInteraction` is a method call and not a field, so it should be @JsonIgnored and not in the document to start with. Where are these fields `interactionIdORgeneMolecularInteraction`, `nameORgeneMolecularInteraction`, and `nameORgeneMolecularInteraction` coming from?

Reply:
@oblodgett The 'OR' is a separator for the fields in order to get the searchDAO query to perform an OR search, so the fields are: 
geneMolecularInteraction.interactionId
geneMolecularInteraction.aggregationDatabase.name
geneMolecularInteraction.interactionSource
geneMolecularInteraction.crossReferences.displayName

Reply:
Ok... I don't remember it working this way... let me take a look

Reply:
Ok seeing the code:
```java
String[] filterNames = filterName.split("OR");
	Arrays.stream(filterNames).forEach(indivFilterName -> {
	BoolQueryBuilder orClause = getBooleanAndedQueryBuilder(indivFilterName.trim(), filterValue);
	outerAndClause.should(orClause);
});
```
Weird... I don't remember us doing this but ok.